pkgs/development/python-modules/pysonos/default.nix

```diff
@@ -29,14 +29,14 @@ buildPythonPackage rec {
   propagatedBuildInputs = [ xmltodict requests ifaddr ];
 
   checkInputs = [
-    pytest pylint flake8 graphviz
+    pytestCheckHook pylint flake8 graphviz
     mock sphinx sphinx_rtd_theme
     requests-mock
   ];
 
-  checkPhase = ''
-    pytest --deselect=tests/test_discovery.py::TestDiscover::test_discover
-  '';
+  disabledTests = [
+    "test_desc_from_uri"
```

Comment:
Add a comment, why this test is deselected and cannot work for us.

Reply:
Thanks, done